src/gui/accountmanager.cpp

```diff
@@ -78,40 +77,45 @@ bool AccountManager::restore()
 bool AccountManager::restoreFromLegacySettings()
 {
     qCInfo(lcAccountManager) << "Migrate: restoreFromLegacySettings, checking settings group"
-             << Theme::instance()->appName();
+                             << Theme::instance()->appName();
```

Comment:
I tried a few things in your branch and it's a bit annoying that clang-format needs to apply the rules no matter what (like we can't seem to tell it to jut not touch the code already there for some criteria). But playing around I think this gives the results I like the most:

```
ColumnLimit: 100
AlignAfterOpenBracket: DontAlign
BinPackArguments: false
```

So I think it would be nice to add `BinPackArguments: false` and I prefer `AlignAfterOpenBracket: DontAlign` over `true` because of the issue we talked above (and some people will continue to indent it manually, I don't think everybody will install clang-format).